### PR TITLE
Apollo Server Project

### PR DESCRIPTION
**Pull request by daniaMaw**
I've edited on the read me file, and added the first project of Apollo server express with GraphQl.

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (NourSammour) [inline .idea/vcs.xml]: this file is related to your IDE, it should not be part from the project
so please add it to `.gitignore`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (NourSammour) [inline package.json:1]: use this package.json
https://github.com/NourSammour/biletiniz/blob/master/package.json

and just change `name` to `biletiniz-api` and keep `scripts` section

**Comment by daniaMaw** [comment]
It all done:
1. index file committed
2. .idea folder is ignored 
3. package file is edited

File: index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+// Construct a schema, using GraphQL schema language
+const typeDefs = gql`
+  type Query {
+    hello: String
+  }
+`;
+
+// Provide resolver functions for your schema fields
+const resolvers = {
+        Query: {
```
Comment (NourSammour) [inline index.js]: move this query to this path
`graphql/resolvers/fake/query/hello.js`

**Comment by daniaMaw** [comment]
I've remover the .idea folder from repo

File: index.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const { ApolloServer, gql } = require('apollo-server-express');
+
+// Construct a schema, using GraphQL schema language
+const typeDefs = gql`
+  type Query {
+    hello: String
+  }
+`;
```
Comment (NourSammour) [inline index.js]: move this type to this path
graphql/schema/fake/index.js

**Comment by daniaMaw** [comment]
- [1 ] Editing the index file
- [2 ] Creating this path graphql/schema/fake/index.js
- [ 3] Creating this path graphql/resolvers/fake/query/hello.js

File: package.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  "name": "biletiniz",
```
Comment (NourSammour) [inline package.json]: your forgot to change name to `biletiniz-api`

**Comment by daniaMaw** [comment]
Sorry fall by mistake >> It's done now

**Pull request merged**